ibm_was/metadata.csv

```diff
@@ -1,48 +1,53 @@
+ibm_was.jdbc.percent_maxed,gauge,,percent,,The percent of the time that all connections are in use.,0,ibm_was,jdbc percent maxed
 ibm_was.jdbc.use_time,gauge,,millisecond,,The average time in milliseconds that a connection is in use.,0,ibm_was,jdbc used time
 ibm_was.jdbc.wait_time,gauge,,millisecond,,The average waiting time in milliseconds until a connection is granted if a connection is not currently available.,0,ibm_was,jdbc wait time
-ibm_was.jdbc.managed_connection_count,gauge,,connection,,"The total number of managed connections in the free, shared, and unshared pools.",0,ibm_was,jdbc managed connection count
-ibm_was.jdbc.connection_handle_count,gauge,,connection,,The number of connections that are in use. Can include multiple connections that are shared from a single managed connection.,0,ibm_was,jdbc connection handle count
-ibm_was.jdbc.prep_stmt_cache_discard_count,gauge,,,,The total number of statements that are discarded by the least recently used (LRU) algorithm of the statement cache.,0,ibm_was,jdbc cache discard count
+ibm_was.jdbc.managed_connection_count,count,,connection,,"The total number of managed connections in the free, shared, and unshared pools.",0,ibm_was,jdbc managed connection count
+ibm_was.jdbc.connection_handle_count,count,,connection,,The number of connections that are in use. Can include multiple connections that are shared from a single managed connection.,0,ibm_was,jdbc connection handle count
+ibm_was.jdbc.prep_stmt_cache_discard_count,count,,,,The total number of statements that are discarded by the least recently used (LRU) algorithm of the statement cache.,0,ibm_was,jdbc cache discard count
 ibm_was.jdbc.jdbc_time,gauge,,millisecond,,"The average time in milliseconds spent running in the JDBC driver that includes time that is spent in the JDBC driver, network, and database.",0,ibm_was,jdbc time
-ibm_was.jvm.heap_size,gauge,,,,The total memory in the JVM run time,0,ibm_was,jvm heap size
-ibm_was.jvm.free_memory,gauge,,,,The free memory in the JVM run time,0,ibm_was,jvm free memory
-ibm_was.jvm.used_memory,gauge,,,,The used memory in the JVM run time,0,ibm_was,jvm used memory
-ibm_was.jvm.up_time,gauge,,,,The amount of time that the JVM is running,0,ibm_was,jvm uptime
-ibm_was.jvm.process_cpu_usage,gauge,,percent,,The CPU Usage (in percent) of the Java virtual machine.,0,ibm_was,jvm process cpu usage
-ibm_was.thread_pools.create_count,gauge,,thread,,The total number of threads created,0,ibm_was,thread pools create count
-ibm_was.thread_pools.destroy_count,gauge,,thread,,The total number of threads destroyed,0,ibm_was,thread pools destroy count
+ibm_was.jvm.heap_size,count,,,,Deprecated use ibm_was.jvm.heap_size_gauge instead,0,ibm_was,jvm heap size
+ibm_was.jvm.free_memory,count,,,,Deprecated use ibm_was.jvm.free_memory_gauge instead,0,ibm_was,jvm free memory
+ibm_was.jvm.used_memory,count,,,,Deprecated use ibm_was.jvm.used_memory_gauge instead,0,ibm_was,jvm used memory
```

Comment:
Just wording, I think adding a comma or a dot after `Deprecated` would make the sentence clearer. With a comma you need to surround the description with quotes to make it work.